app/models/docker_container_wizard_states/exposed_port.rb

```diff
@@ -0,0 +1,15 @@
+module DockerContainerWizardStates
+  class ExposedPort < Parameter
+    Parameter.reset_callbacks(:validate)
```

Comment:
Why? This and the other call to `reset_callbacks` are what is making tests fail, because the Parameter class validations go away after that.